wiki/Contests/TKBC/2/ko.md

```diff
@@ -0,0 +1,75 @@
+---
+tags:
+- TKBC
+- TKBC2
+---
+
+# Taiko Korean Beatmap Contest #2
+
+![TKBC2 logo](img/logo.png)
+
+**Taiko Korean Beatmap Contest #2** (***TKBC2***)는 ![][flag_KR]한국인을 위한 태고 대회입니다. Taiko Korean Beatmap Contest 2회차에 대한 내용을 담고 있습니다.
+
+## 대회 일정
+
+| 일정 | 날짜 (UTC+09) |
+| --: | :-- |
+| 제출 기간 | 2020-11-30/2020-12-27 |
+| 심사 기간 | 2020-12-28/2021-01-17 |
+| 결과 발표 | 2021-01-17 |
+
+## 주최
+
+Taiko Korean Beatmap Contest #2는 한국 태고 매핑 커뮤니티 인원들에 의해 주최됩니다.
+
+| 직책 | 인원 |
+| :-- | :-- |
+| Manager | ![][flag_KR] [\[Karcher\] 2](https://osu.ppy.sh/users/9892196), ![][flag_KR] [Civil oath](https://osu.ppy.sh/users/3216107) |
+| Designer | ![][flag_KR] [oL0](https://osu.ppy.sh/users/1134683) |
+| Judge | ![][flag_KR] [\[Karcher\] 2](https://osu.ppy.sh/users/9892196), ![][flag_KR] [Petit](https://osu.ppy.sh/users/4637369), ![][flag_KR] [Ozu](https://osu.ppy.sh/users/980092) |
+
+## 상품
+
+| 순위 | 상품 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 프로필 뱃지 |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 서포터 2개월 |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 서포터 1개월 |
+
+## 링크
+
+- [포럼 스레드](https://osu.ppy.sh/community/forums/topics/1162734)
+
+## 결과
+
+| 순위 | 매퍼 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_KR] [applerss](https://osu.ppy.sh/users/983349) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_KR] [4\_4](https://osu.ppy.sh/users/1152851) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_KR] [ddm](https://osu.ppy.sh/users/7910282) |
```

Comment:
the alt texts and tooltips of images have to be translated

for reference:

`![alt-text](image-path "tooltip")`